lib/npm-check-updates.js

```diff
@@ -121,7 +121,8 @@ function upgradePackageDefinitions(currentDependencies) {
 
     return vm.getLatestVersions(currentDependencies, {
         versionTarget: versionTarget,
-        registry: options.registry ? options.registry : null
+        registry: options.registry ? options.registry : null,
+        includePrerelease: options.pre
```

Comment:
Didn't notice this before, but let's stick to using the same name for command line options and internal options. It's confusing to suddenly switch from `pre` to `includePrelease`. The README and binary provide enough explanation of what `pre` does for it to be understandable deeper in the code.